### ISSUE TITLE
Unable to Install Specto Fork on any Kodi version, after a Win 7 complete re-installation. Assitance, please!

### ISSUE THREAD

**Issue by Allexxiell13** [mrknow/filmkodi#937]
## Please follow the guide below 
- Put an "x" into all boxes [ ] relevant to your *issue*
- Use *Preview* tab to see how your issue will actually look like
---
## Kodi & Addon 
- Kodi verison: from 14.0 Helix to 17.6 Krypton
- [ ] Filmy online - Mrknow
- [ ] MrknowTV
- [x] Specto
- [ ] FanFILM
- Version / Wersja: Windows 7 Professional SP1, x64, DELL Studio 1535 laptop
---
Hello to everyone :)

So I had to re-install my Win 7 Professional 3 days ago due to problems and afterwards, no matter which version on Kodi I installed (believe me tried at east 7 different from the Helix to the Krypton one) and it is not working.
For example, regarding the 17.6 Krypton even after enabling the Unknown Sources, and carefully checking the address for mistakes, and typing a different one, I still received the: "Unable to connect to the server. Check the network connection, etc." When I tried with the Isengard, Jarvis and Helix versions (with teh repsected name for the Specto repository to appear) it is still not working. I see it in add-ons, however when I go to "Install from .zip file" the file is Empty! Literally! Or simply does not open, hahha this is funny :P.
Once again, before I formatted and re-installedd the same Win 7 version on my DELL laptop -everything was perfect! I was using the Jarvis verison. My Internet connetcion is excellent, the addresses I am typing are correct, Windows finished installing the most important updates, etc.... 
I don't understand why it is still failing to install (having in mind that last Friday it was working just fine). Is it possible that I am missing a yet-to-be-installed-unpdate for Win 7? Could it be that the server is temporarily down for some reason?
I have currently un-installed Kodi.

Any advices and help would be appreciated.
Thank you in advance! :)

**Comment by BankruptEd** [comment]
Are you sure Specto is even being maintained? Maybe the fact it's been
dying a slow death is part of the problem. Try Exodus. It seems to be
maintained still, even if it's not by the original guy.

**Comment by Allexxiell13** [comment]
Hi and thank you for the reply :).
I tried multiple addons, Exodus included. 
I even tried installing the Ares Wizard and it was the only one thatgot activated! 
However!
When I open it the first 2 options, like "Browse addons (from where I can choose between a multiple of them)" are in gray and inactive :(. I read there is a lawsuit happening in the US with the Kodi authors, and almost everything is off since 2 months.
I guess it is either with me or there sure is a major problem with the program... *sigh*.

**Comment by BankruptEd** [comment]
Yes. I heard that Area was kaput as well cause of the lawsuits and what
not. It's sad. I'm waiting for the much anticipated new add-ons from Indigo
/ TVAddons. It's pretty dry out there right now. I basically live in
Terrarium TV, although there are a few Kodi add-ons I still use. Good luck
my friend :)
